ui/src/app/kvm/views/KVMList/AddKVMForm/AddKVMForm.tsx

```diff
@@ -28,7 +28,9 @@ import type { PowerType } from "app/store/general/types";
 import type { TSFixMe } from "app/base/types";
 import zoneSelectors from "app/store/zone/selectors";
 
-const generateAddKVMSchema = (parametersSchema: TSFixMe) =>
+const generateAddKVMSchema = (
+  parametersSchema: Yup.ObjectSchemaDefinition<TSFixMe>
```

Comment:
As this is a dynamic schema, I'm not sure how to type it...

Reply:
Yeah I can't see how this is supposed to be done either.